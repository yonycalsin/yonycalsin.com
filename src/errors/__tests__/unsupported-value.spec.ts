import UnsupportedValueError from '../unsupported-value'

describe('Errors / UnsupportedValueError', () => {
  it('throws unsupported value error', () => {
    const fn = () => {
      throw new UnsupportedValueError("The draft status does'nt exists")
    }

    expect(() => fn()).toThrowErrorMatchingSnapshot()
  })
})
