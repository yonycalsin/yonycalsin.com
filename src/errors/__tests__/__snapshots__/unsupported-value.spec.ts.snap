// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors / UnsupportedValueError throws unsupported value error 1`] = `"Unsupported value: The draft status does'nt exists"`;
